app/javascript/components/applications/application_form/DemographicInfoSection.js

```diff
@@ -10,7 +11,13 @@ let {
   ethnicityOptions
 } = formOptions
 
-const DemographicInfoSection = ({ defaultValues }) => {
+const DemographicInfoSection = ({ values: { demographics } }) => {
+  const validateRelatedField = (validationMessage, relatedField) => {
```

Comment:
@LoganArnett this actually isn't the desired behavior. It's more like gender_other is required if the leasing agent selects "not listed" for gender. That's an optional piece of this ticket though, don't need to implement it right now.

Reply:
I missed that that was an option in the `gender` field I can update that if we want for this PR

Reply:
ok yeah sounds good! then it's just a refactor of what you have already, right?

Reply:
Added this in